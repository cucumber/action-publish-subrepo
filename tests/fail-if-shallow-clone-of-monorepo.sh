#!/bin/bash
set -e

pushd monorepo || exit
mkdir ruby
echo "# Ruby project" >ruby/README.md
git add .
git commit --message "Add ruby"

mkdir php
echo "# PHP project" >php/README.md
git add .
git commit --message "Add php"
popd || exit

# make a shallow clone and move into its folder
git clone --depth 1 "file://$(realpath monorepo)" "$(realpath monorepo-clone)" --quiet
pushd monorepo-clone || exit

# run the tool
publish-to-subrepo ruby "$SUBREPO"
