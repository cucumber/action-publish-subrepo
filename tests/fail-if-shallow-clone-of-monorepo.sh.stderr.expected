This is a shallow clone! Need full depth.
